.github/workflows/ibis-docs-main.yml

```diff
@@ -69,10 +69,18 @@ jobs:
         run: |
           python -m pip install --upgrade algoliasearch
 
-      - name: Create and Upload Index
+      - name: Create and Upload Base Index
         run: |
           python .github/workflows/upload-algolia.py
         env:
           ALGOLIA_WRITE_API_KEY: ${{ secrets.ALGOLIA_WRITE_API_KEY }}
           ALGOLIA_APP_ID: HS77W8GWM1
           ALGOLIA_INDEX: prod_ibis
+
+      - name: Create and Upload API Records to index
+        run: |
+          python .github/workflows/upload-algolia-api.py
+        env:
+          ALGOLIA_WRITE_API_KEY: ${{ secrets.ALGOLIA_WRITE_API_KEY }}
+          ALGOLIA_APP_ID: HS77W8GWM1
+          ALGOLIA_INDEX: prod_ibis
```

Comment:
Is there a way to avoid duplicating these envs across steps? Like a top-level `env` mapping? Not a big deal.

Reply:
probably, I'll try to consolidate in a follow-up when i add a few more tweaks to the algolia index creation.